docs/localization.rst

```diff
@@ -67,25 +66,25 @@ Adding a new Locale
 
 #. Create the `jsi18n` file for the new locale::
 
-        $ ./manage.py compilejsi18n
+        make compilejsi18n
 
 #.  Verify django loads new locale without errors by visiting the locale's home
     page. E.g., https://developer-local.allizom.org/ml/
 
 #.  BONUS: Use `podebug` to test a fake translation of the locale::
 
-        $ cd locale
-        $ podebug --rewrite=bracket templates/LC_MESSAGES/django.pot ml/LC_MESSAGES/django.po
-        $ ./compile-mo.sh .
+        cd locale
+        podebug --rewrite=bracket templates/LC_MESSAGES/django.pot ml/LC_MESSAGES/django.po
+        ./compile-mo.sh .
```

Comment:
It looks like there's a bug in plural forms:

```
#: kuma/search/jinja2/search/results.html:88
#, python-format
msgid "%(count)s document found in %(locale_native)s."
msgid_plural "%(count)s documents found in %(locale_native)s."
msgstr[0] ""
msgstr[1] ""
msgstr[0] "%(count)s document found in %(locale_native)s ye scalleywag❗."
msgstr[1] "%(count)s documents found in %(locale_native)s cap'n❗."
```

This causes an error when compiling to .mo files. If you remove the blank lines, it works:

```
#: kuma/search/jinja2/search/results.html:88
#, python-format
msgid "%(count)s document found in %(locale_native)s."
msgid_plural "%(count)s documents found in %(locale_native)s."
msgstr[0] "%(count)s document found in %(locale_native)s ye scalleywag❗."
msgstr[1] "%(count)s documents found in %(locale_native)s cap'n❗."
```

Still, fun.